h2o-core/src/main/java/water/RPC.java

```diff
@@ -175,64 +175,59 @@ private void handleCompleter( CountedCompleter cc ) {
 
     // Keep a global record, for awhile
     if( _target != null ) _target.taskPut(_tasknum,this);
-    try {
```

Comment:
I would keep as-is

Reply:
this change has 2 parts:
1) remove try..catch which IMHO is a NOOP
t.printStackTrace();
      throw Log.throwErr(t); // this prints the exception again and re throws it as runtimeexception
2) whitespace change because of remove block

Reply:
ad (1) - it is not NOOP, it is attempting to make sure the Throwable is written at least to stdout and logged. This is a desperate attempt to preserve the exception (even such as OutOfMemoryError). This is done on purpose - I would leave as-is.